diff -up sfxr-sdl/sdlkit.h~ sfxr-sdl/sdlkit.h
--- sfxr-sdl/sdlkit.h~	2008-11-13 17:25:52.000000000 +0100
+++ sfxr-sdl/sdlkit.h	2008-11-13 17:31:05.000000000 +0100
@@ -128,6 +128,7 @@ static void sdlquit ()
 static void sdlinit ()
 {
 	VERIFY(!SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO));
+	SDL_WM_SetIcon(SDL_LoadBMP("/usr/share/sfxr/sfxr.bmp"), NULL);
 	atexit(sdlquit);
 	memset(keys, 0, sizeof(keys));
 	ddkInit();
